.github/workflows/dev.yml

```diff
@@ -9,3 +9,8 @@ on:
 jobs:
   build:
     uses: axonivy-market/github-workflows/.github/workflows/dev.yml@v4
+    with:
+      ivyPluginVersion: 11.3.0
+      testerVersion: 11.3.0
+      ivyVersion: nightly-11.3
```

Comment:
there is a reproducible build error when running with 11.4 ; going to investigate later on ... for now we downgrade to the public LE release 11.3